modules/ROOT/pages/pcf-space-config.adoc

```diff
@@ -61,9 +57,9 @@ cf marketplace
 
 === pcf-cloud-params
 
-The Space Configuration Tool uses the properties defined in `pcf-cloud-params` to register a space in an Anypoint Platform organization. Using these properties, it makes API calls directly to the Anypoint Runtime Manager.
+The Space Configuration Tool uses the properties defined in `pcf-cloud-params` to register a space in an Anypoint Platform organization.
 
-Configure each of the following properties in the `space_settings.yml`. The values you define here must be unique for each PCF space you are configuring.
+Configure each of the following properties in the `space_settings.yml`.
```

Comment:
Unsure if all of these properties still need to be defined; need engineering to comment.